pkg/controllers/termination/controller.go

```diff
@@ -36,6 +36,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+// Now is a time.Now() that may be mocked by tests.
+var Now = time.Now
```

Comment:
Do we need this if we have `utils/injectabletime`?

Reply:
Thanks for this!